.circleci/config.yml

```
@@ -66,10 +66,10 @@ jobs:
           path: /tmp/test-results
           destination: test-results
 
-  build-rails-6-0:
+  build-rails-6-1:
```

This should probably say something like this: 

```suggestion
  build-rails-next:
```